files/scripts/swss.sh

```diff
@@ -206,10 +206,13 @@ SERVICE="swss"
 PEER="syncd"
 DEBUGLOG="/tmp/swss-syncd-debug$DEV.log"
 LOCKFILE="/tmp/swss-syncd-lock$DEV"
+NAMESPACE_PREFIX="asic"
 if [ "$DEV" ]; then
-    NET_NS="asic$DEV" #name of the network namespace
+    NET_NS="$NAMESPACE_PREFIX$DEV" #name of the network namespace
+    SONIC_DB_CLI="sonic-db-cli -n $NET_NS"
 else
-    NET_NS=""   
+    NET_NS=""
+    SONIC_DB_CLI="sonic-db-cli"
```

Comment:
>SONIC_DB_CLI="sonic-db-cli" [](start = 4, length = 27)

Why not merge the two branches into a unified.
What is the gap to use `sonic-db-cli -n ''` directly?

Reply:
As you know we made this change in sonic-db-cli for the user to use either a <valid namespace> don't pass any if there is no namespace ( in which case it will be default to None ). Yes I agree, passing '' will work as we know the internal implementation of SonicDBConfig.. but it could confuse other code readers. This was the idea behind this approach.

Reply:
Thanks for confirming it will work. No need to worry about the readability, just add a big chunk of messages to explain what happens in sonic-db-cli usage. I believe this will simplify code a lot.

---
In reply to: [414899788](https://github.com/Azure/sonic-buildimage/pull/4477#discussion_r414899788) [](ancestors = 414899788)

Reply:
@lguohan Please could you share your thoughts on the preferable approach ? 

Shall we allow the user to pass **the option -n ''** to the scripts like sonic-db-cli, sonic-cfggen ? Passing '' will internally refer to the local database_config.json file.

The other option to get this done would be to not pass the -n option itself. If the user wants to refer to the local database_config.json file he **will not give the -n option**, which will result in namespace=None begin passed internally to SonicV2Connector and mapped to the local database_config.json file.